src/build-from-oxlint-config.ts

```diff
@@ -112,34 +157,40 @@ const handleRulesScope = (
   rules: Record<string, 'off'>
 ): void => {
   for (const rule in oxlintRules) {
+    const eslintName = getEsLintRuleName(rule);
+
+    if (eslintName === undefined) {
+      // ToDo: output?
```

Comment:
oxlint is already outputting it. should we do the same too?

Reply:
Yes we should warn about it.

Reply:
done